service.go

```diff
@@ -201,7 +203,7 @@ type Builder struct {
 // New initiates the Service builder chain.
 // The builder contains default values for Alive/Ready checks,
 // the SIGHUP handler and its version.
-func New(name, version string) *Builder {
+func New(name, version, env string) *Builder {
```

Comment:
Since we are using the builder pattern, we could just add a new `WithEnvironment` in order to make this optional.